### ISSUE TITLE
Nightly (2023-02-22) breaks DownThemAll interaface

### ISSUE THREAD

**Issue by jiangzhenjerry** [xiaoxiaoflood/firefox-scripts#246]
Today (2/22/2023) Firefox Nightly breaks the interface of DownThemAll again. It looks funny (but still functional!)

<img width="741" alt="sshot-1" src="https://user-images.githubusercontent.com/9311082/220730954-6c479806-24ba-4588-8652-70b45fd3b1a0.png">

**Comment by jiangzhenjerry** [comment]
OK things are getting worse today... I cannot see the window control (min/max/close) anymore...

<img width="759" alt="2023-02-23_160531" src="https://user-images.githubusercontent.com/9311082/221059550-2648ab55-8a21-414d-879c-bbeb91367759.png">

**Comment by marty60** [comment]
They also broke the Password Manager interface again, it's like the fourth time in the last six months.

**Comment by TimidScript** [comment]
I have the same issue on a clean install on FF 112. However on another profile that I had it installed for months it still works fine. Just the new install is all corrupt, 


![Snap1](https://user-images.githubusercontent.com/8114076/231440845-a9affb21-caee-4aa6-94d0-908fd305ac5f.jpg)

**Comment by dimdamin** [comment]
moz-image-region removed in [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1817071) from ff code, but nor remover from HTML5
[DownThemAll-3.1.2.112.2023.04.22-xiao.xpi.zip](https://github.com/xiaoxiaoflood/firefox-scripts/files/11321750/DownThemAll-3.1.2.112.2023.04.22-xiao.xpi.zip)

**Comment by jathek** [comment]
> moz-image-region removed in [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1817071) from ff code, but not removed from HTML5 [DownThemAll-3.1.2.112.2023.04.22-xiao.xpi.zip](https://github.com/xiaoxiaoflood/firefox-scripts/files/11321750/DownThemAll-3.1.2.112.2023.04.22-xiao.xpi.zip)

Anyone else having issues resizing columns in the download manager? Some of the columns refuse to shrink and it's pushing the scrollbar out of the window.

Small missed icon, I think:
![image](https://user-images.githubusercontent.com/692914/234355280-c702736f-e89a-49a8-baf6-365594bf9ce1.png)

Overall great fix, thanks.

**Comment by jiangzhenjerry** [comment]
> Anyone else having issues resizing columns in the download manager? Some of the columns refuse to shrink and it's pushing the scrollbar out of the window.

Yeah... I'm also observing squeezed columns that cannot be expanded.

<img width="509" alt="sshot-1" src="https://user-images.githubusercontent.com/9311082/234361477-8a7f3b51-af22-425f-bf54-36f58959467c.png">

**Comment by xelsios** [comment]
Not only moz-image-region got removed, also multiple other moz-box-* CSS properties will be removed in [v113](https://bugzilla.mozilla.org/show_bug.cgi?id=1824957) which will break the preferences panel completly.
My try on fixing it takes a different route with handling the moz-image-region by splitting then png in multiple files, but the result is the same working extension. 
Also the mentioned moz-box properties are handled (but only on win, mac und unix are not fixed and not tested).
[dta.xpi.zip](https://github.com/xiaoxiaoflood/firefox-scripts/files/11325705/dta.xpi.zip)

**Comment by TFWol** [comment]
@xelsios Good change, thank you. Looks like it was annoying to do.

**Comment by dimdamin** [comment]
I updated the interface in DownThemAll and S3Download, works for me... Maybe it will be works for you to, tested at 102 and 113
[DownThemAll-3.1.2.112.2023.05.10-xiao.xpi.zip](https://github.com/xiaoxiaoflood/firefox-scripts/files/11449117/DownThemAll-3.1.2.112.2023.05.10-xiao.xpi.zip)
[s3download-4.14.109.2023.05.10-xiao.xpi.zip](https://github.com/xiaoxiaoflood/firefox-scripts/files/11449136/s3download-4.14.109.2023.05.10-xiao.xpi.zip)

**Comment by marty60** [comment]
> Not only moz-image-region got removed, also multiple other moz-box-* CSS properties will be removed in [v113](https://bugzilla.mozilla.org/show_bug.cgi?id=1824957) which will break the preferences panel completly. My try on fixing it takes a different route with handling the moz-image-region by splitting the PNG images in multiple files, but the result is the same working extension. Also the mentioned moz-box properties are handled (but only on win, mac und unix are not fixed and not tested). [dta.xpi.zip](https://github.com/xiaoxiaoflood/firefox-scripts/files/11325705/dta.xpi.zip)

Will these changes have any effect on fixing the password manager interface?

**Comment by marty60** [comment]
dimdamin,

Thanks for the DTA interface fix, that was nasty. I also saw where you fixed the columns in the DownThemAll interface in select.xhtml with flex-grow:. I was able to apply that to password manager with success.

**Comment by marty60** [comment]
Thanks all for the DTA interface fix, that was nasty. I also saw where the columns in the DownThemAll interface were fixed in select.xhtml with flex-grow:. I was able to apply that to password manager with success.

**Comment by jiangzhenjerry** [comment]
Thanks @xelsios and @dimdamin for the fixes. Wanted to give all the DTA fans an update that DTA is completely dead under 115a1 starting 2023/05/13. I started a new bug report #259. Hope there will be a fix eventually.

**Comment by TFWol** [comment]
I'm worried about @xiaoxiaoflood ; no activity on github for a couple of months.

**Comment by marty60** [comment]
> I'm worried about @xiaoxiaoflood ; no activity on github for a couple of months.

He's around, I think he posts at Reddit as tustamido and there's new ones today. 

https://old.reddit.com/r/XUL_for_Quantum_Dev/comments/fs76ff/little_update_current_fx_version_75/
